lib/Runtime/Math/AsmJsMath.inl

```diff
@@ -167,4 +167,27 @@ namespace Js
         }
         return 32;
     }
+
+    inline int AsmJsMath::Eqz(int value)
+    {
+        return value == 0;
+    }
+
+    template<typename T>
+    inline T AsmJsMath::Copysign(T aLeft, T aRight)
+    {
+        return copysign(aLeft, aRight);
+    }
+
+    template<typename T>
+    inline T AsmJsMath::Trunc(T value)
+    {
+        return value;
```

Comment:
can you also Assert(UNREACHED) here and in Nearest? If you're not going to implement these yet, it's important that we have explicit failure in case someone tries to use them.